frontend/src/shared/modules/filters/components/partials/FilterIncludeSwitch.vue

```diff
@@ -1,12 +1,19 @@
 <template>
-  <div class="flex justify-start border-b border-gray-100 p-4">
+  <div class="flex justify-start items-center border-b border-gray-100 p-4">
     <el-switch
       v-model="model"
       :active-text="modelValue ? 'Include' : 'Exclude'"
       class="w-fit flex !justify-start !m-0 filter-include-switch"
       size="small"
       data-qa="filter-include-switch"
     />
+
+    <el-tooltip
+      content="If excluded, turns the filter into a negative condition"
+      placement="top"
+    >
+      <i class="ri-question-line text-gray-400" />
```

Comment:
Can you also add a class "text-base"? That wasn't on the specification, but the icon should be slightly bigger 🙏